cas-server-webapp/src/main/webapp/WEB-INF/view/jsp/default/ui/includes/top.jsp

```diff
@@ -99,6 +99,10 @@
                             <c:when test="${osfLoginContext.isInstitutionLogin()}">
                                 <spring:message code="screen.institution.login.message" />
                             </c:when>
+                            <c:when test="${osfLoginContext.isUnsupportedInstitutionLogin()}">
```

Comment:
To myself: it's fine here since the backend guarantees that `.isInstitutionLogin()` and `.isUnsupportedInstitutionLogin()` is mutually.